src/awkward/_util.py

```diff
@@ -31,6 +29,36 @@
 kSliceNone = kMaxInt64 + 1  # for Slice::none()
 kMaxLevels = 48
```

Comment:
Maybe you meant de-duplicating with this copy of constants? (Note the comment: `# matches include/awkward/common.h`.)

Reply:
Could you clarify, is it just my short-hand that needs elaborating here? I.e. 
> Duplicated constants from `include/awkward/common.h`

We could also export them if we needed.